### DISCUSSION TITLE
open board lag when it is used with OBS

### DISCUSSION THREAD

**Discussion by vivshankh** [community/community#142170] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

open board lag when it is used with OBS

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @vivshankh ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category and including relevant details. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by luannndev** [comment]
Hi vivshankh,

It sounds like you're experiencing lag issues when using Open Board with OBS (Open Broadcaster Software). Here are some potential solutions to help mitigate the lag:

Lower Resolution: Reduce the resolution of the Open Board display in OBS. A lower resolution can help decrease the load on your system and improve performance.

Adjust OBS Settings: Try lowering the frame rate and bitrate settings in OBS. This can help reduce the processing required for streaming.

Close Background Applications: Ensure that no unnecessary applications are running in the background while you're using Open Board and OBS. This can free up system resources.

Use Hardware Acceleration: Check if OBS and Open Board have hardware acceleration options available. Enabling these can improve performance by utilizing your GPU instead of relying solely on the CPU.

Update Software: Make sure that both OBS and Open Board are up to date. Software updates often include performance improvements and bug fixes.

Optimize Graphics Settings: If you are using a graphics-intensive application alongside OBS, consider lowering the graphics settings in that application to reduce overall strain on your system.

If the problem persists, you may want to check the OBS forums or community support channels for further assistance.
